bin/civihr-install

```diff
@@ -596,8 +609,7 @@ export PATH="$PATH:$BINDIR"
 civihr_parse "$@"
 
 # Checking for Drush dependencies if not present downloading into temp.
-command -v drush >/dev/null 2>&1 || ( echo "[[Downloading dependencies.........]]" && curl -Lo "$TMPDIR/drush" "https://github.com/drush-ops/drush/releases/download/8.1.8/drush.phar"  && chmod +x "$TMPDIR/drush" && export PATH="$PATH:$TMPDIR/drush" )
-
+command -v drush >/dev/null 2>&1 || ( echo "[[Downloading dependencies.........]]" && curl -Lo "$TMPDIR/drush" "https://github.com/drush-ops/drush/releases/download/8.1.8/drush.phar" && chmod +x "$TMPDIR/drush" && export PATH="$TMPDIR:$PATH" )
```

Comment:
As an user, I would be VERY mad to find out a script modified my PATH without at least asking me if I'm ok with that.

I think we should either:
- Make drush mandatory. So we just inform the user that drush has to be installed and available and stop the script if it's not
- Tell the user that drush is not present and ask if it's ok to download it and add it to PATH
- Download it silently, but instead of adding it to PATH, add the absolut path to a variable (DRUSH=$TMPDIR/drush) and then use it to call drush. I'm not sure if that is possible though